mysql-test/suite/innodb/t/percona_bug_ps8428.test

```diff
@@ -0,0 +1,11 @@
+#
```

Comment:
we can rename the test without a bug number. percona_encrypt_online_alter?

Reply:
There is already `innodb.innodb_row_log_encryption` MTR test case which tests `--innodb_encrypt_online_alter_logs=ON` and originally I wanted to extend it instead of creating a new one.
However, it has different `.opt` file and is shared with another test from `keyring_vault` via `.inc` file.
So, I'd rather leave it as is.